README.md

```diff
@@ -57,7 +57,7 @@ Add a build step using the test-summary plugin:
 ```yaml
   - label: annotate
     plugins:
-      - bugcrowd/test-summary#v1.11.0:
+      - bugcrowd/test-summary#v1.13.0:
```

Comment:
Should this be 
```suggestion
      - instacart/test-summary#v1.13.0:
```
?

Reply:
Ah right, not sure if we intend on merging this back in upstream or should just maintain this as a separate branch